documentation/config_file_auth.md

```diff
@@ -159,10 +185,71 @@ Puppet Server allows agents to make requests at the old URLs and internally tran
-
 For backward compatibility, settings in [`puppetserver.conf`][] also control whether to use the new Puppet Server authorization method for certain endpoints:
 
 * `use-legacy-auth-conf` in the `jruby-puppet` section: If `true`, Puppet Server uses the Ruby authorization methods and  [Puppet `auth.conf`][] format and warns you that this is deprecated. If `false`, Puppet Server uses the new authorization method and HOCON `auth.conf` format. Default: `true`.
 * `authorization-required` and `client-whitelist` in the `puppet-admin` section: If `authorization-required` is set to `false` or `client-whitelist` has at least one entry, Puppet Server authorizes requests to Puppet Server's administrative API according to the parameters' values. See the [`puppetserver.conf` documentation][`puppetserver.conf`] for more information on these settings. If `authorization-required` is set to `true` or not set and `client-whitelist` is set to an empty list or not set, Puppet Server authorizes requests to Puppet Server's administrative API using the authorization method introduced in Puppet Server 2.2.
 * `certificate-status.authorization-required` and `certificate-status.client-whitelist` in the `certificate-authority` section: If `authorization-required` is set to `false` or `client-whitelist` has one or more entries, Puppet Server handles requests made to its [Certificate Status](/puppet/latest/reference/http_api/http_certificate_status.html) API according to the parameters' values. See the [`ca.conf` documentation](./config_file_ca.html) for more information on these settings. If `authorization-required` is set to `true` or not set and the `client-whitelist` is set to an empty list or not set, Puppet Server authorizes requests using the authorization method introduced in Puppet Server 2.2.
+
+## Deprecated Ruby Parameters
+
+> **Deprecation Note:** The legacy [Puppet `auth.conf`][] rules for the master endpoints, and client whitelists for the Puppet admin and certificate status endpoints, are deprecated. Convert your configuration files to the HOCON formats using the equivalent [HOCON parameters](#hocon-parameters).
+
+### `path`
+
+Rules with a `path` parameter apply only to endpoints with URLs that start with the parameter's value. In the deprecated Puppet `auth.conf` format, start the `path` value with a tilde (`~`) character to indicate that it contains a regular expression.
+
+~~~
+# Regular expression to match a path in a URL.
+path ~ ^/puppet/v3/report/([^/]+)$
+
+# Literal string to match at the start of a URL's path.
+path /puppet/v3/report/
+~~~
+
+### `method`
+
+If a rule contains the `method` parameter, it only applies to requests that use the value's corresponding HTTP methods. In the deprecated `auth.conf` format, use indirector names for the `method` value:
+
+Indirector | HTTP
```

Comment:
As a further example of the "migrating your settings..." approach, this table's column names could have `Deprecated Ruby auth.conf` vs. `New Puppet Server auth.conf` - or something similar.  The point would be to show users that where they had to use "indirector" based abstractions, which could be roughly translated into corresponding HTTP methods, in the deprecated Ruby auth.conf format, users would instead need to express the HTTP method names directly in the new Puppet Server auth.conf format.